docs/source/notebooks/Model_calibration.ipynb

```diff
@@ -170,7 +170,7 @@
    "name": "python",
    "nbconvert_exporter": "python",
    "pygments_lexer": "ipython3",
-   "version": "3.8.6"
+   "version": "3.7.3"
```

Comment:
Why the python downgrade here?

Reply:
Eh.. not sure, I've too many environments...

Reply:
Unfortunately, I don't think we can strip this information easily. `nbstripout` removes a lot but this seems to keep getting updated. We can probably just rollback these changes when they happen.